x/orderbook/codec.proto

```diff
@@ -56,14 +56,14 @@ message Order {
   coin.Coin original_offer = 7;
   // Remaning offer. (This part enables partially filled orders)
   coin.Coin remaining_offer = 8;
-  // Price for each unit (in tickers of the opposite currency from our side)
+  // Price defines how many units of Bid we will offer for one unit of Ask
+  // If Side == Ask, we sell ask and accept any price HIGHER
+  // If Side == Bid, we buy ask and accept any price LOWER
   Amount price = 9;
-  // Executed trades (referenced ids)
-  repeated bytes trade_ids = 10;
```

Comment:
I see you removed trade id's that are executed under an order. May I ask how we will query which trades executed under an order?

Reply:
Good question:

Trade Bucket has an index by order. You can query that. No need to keep the data in three places (the index, and both orders participating in the trade). ORM guarantees index is always up to date.